core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -73,6 +77,21 @@
     UNKNOWN,
   }
 
+  /** Use to filter property keys that start with alluxio.*/
+  private static final String ALLUXIO_CONF_PREFIX = "alluxio";
+  /** Use to filter property keys that start with aws.*/
+  private static final String AWS_CONF_PREFIX = "aws";
+  /** Use to filter property keys that start with fs.*/
+  private static final String FS_CONF_PREFIX = "fs";
+  /** Prefixes that are not related to workers but exists in worker-related scope.*/
+  private static final String[] WORKER_NOT_RELATED_PREFIX = {"alluxio.fuse", "alluxio.proxy",
+      "alluxio.integration.master", "alluxio.integration.mesos.master"};
+  /** Properties that are not related to workers but exist in worker-related scope.*/
```

Comment:
If these are not supposed to be in the worker's scope, then shouldn't we just remove them from the worker scope?

Reply:
Those properties have the range Scope.ALL.  For Scope=worker(Property is used in workers), Scope=SERVER(Property is used in masters and workers) we are clear that those properties are related to workers. 

But for Scope=ALL which is the default scope, many properties are integration related properties that may/may not related to workers.

Reply:
If a property does not apply to workers and its scope is set to `Scope.ALL`, the scope is set incorrectly and should be fixed. Mesos integration properties are not used by clients or servers, so they should have `Scope.NONE`. The master properties only apply to master, so they should have scope `Scope.MASTER`